### PR TITLE
Change to SD MadSkillz to align with skills and capabilities expected…

### PR DESCRIPTION
**Pull request by mandyfegredo**
… for a promotion to SC

### CODE REVIEW

**Comment by Meligy** [comment]
Achievement unlocked:  First PR created
Next Achievement: First PR approved

File: Consulting.md [hunk 1]
```diff
@@ -61,11 +63,16 @@
 - I adapt quickly to changing requirements and typically structure my approach to cater for this.
 - I have proven I can decompose larger requirements into smaller, more manageable pieces, to deliver functionality in an incremental and iterative manner.
 - I possess writing skills that let me be clear, concise, articulate and persuasive in all of my outgoing communications.
```
Comment (andrewabest) [inline Consulting.md]: The two points immediately below this are covered pretty well under the heading *I am an egoless, team oriented developer* in the developer MadSkillz.

File: Consulting.md [hunk 2]
```diff
@@ -50,6 +51,7 @@
 - I use code reviews as an opportunity to teach and show others alternate, cleaner ways to implement functionality in an ego-less manner. That way the whole team together learns how to deliver a higher quality, more maintainable product.
 - I can tactfully challenge and coach people and have them leave richer for the experience.
 - I am a confident and proficient teacher of my craft.
```
Comment (andrewabest) [inline Consulting.md]: This one is also covered under the the heading *I am an egoless, team oriented developer* in the developer MadSkillz I think

> I communicate professionally and keep stake holders engaged at all times.

and below this we have 

> I possess writing skills that let me be clear, concise, articulate and persuasive in all of my outgoing communications

Could this one be more specific potentially?

**Comment by andrewabest** [comment]
Are these SD behaviors then, or SC behaviors @mandyfegredo ? To be promoted to SC you need to both be 'no brainer' in all SD behaviours, and be exhibiting and actively practicing some of the SC behaviors.

In particular I see:
* I participate in improving Readify processes.
* I am proficient at qualifying and mitigating risk.

Potentially as SC behaviours.

File: Consulting.md [hunk 3]
```diff
@@ -61,11 +63,16 @@
 - I adapt quickly to changing requirements and typically structure my approach to cater for this.
 - I have proven I can decompose larger requirements into smaller, more manageable pieces, to deliver functionality in an incremental and iterative manner.
 - I possess writing skills that let me be clear, concise, articulate and persuasive in all of my outgoing communications.
+- I Interact well with my team mates and our customers.
+- My team find me easy to work with.
+- I participate in improving Readify processes.
```
Comment (andrewabest) [inline Consulting.md]: I'd say this one could be included in the **I've become a student of my craft.** characteristic below.

File: Consulting.md [hunk 4]
```diff
@@ -61,11 +63,16 @@
...
+- I readily share my knowledge with fellow Readifarians and client developers.
 
 #### I've become a student of my craft.
 - I am devoted to learning; it's become a natural part of my life.
 - I am actively introspective and take time to work on my personal and professional development.
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
```
Comment (andrewabest) [inline Consulting.md]: There is 

> I can tactfully challenge and coach people and have them leave richer for the experience.

Under the **I am a technical mentor.** characteristic. Does that cover this well enough?

File: Consulting.md [hunk 5]
```diff
@@ -61,11 +63,16 @@
...
+- I actively Challenge assumptions and participate in healthy debates to find the best solution for the situation.
```
Comment (kjacobsen) [inline Consulting.md]: This probably should read:

> I actively challenge assumptions and participate in discussions to find the best solution for the situation.

1. Changed capitalization.
2. Health debates implies there are a) unhealthy ones occurring within the organization and b) that it is more formal argumentative process.

File: Consulting.md (hunk 3 above)
Comment (kjacobsen) [inline Consulting.md]: Agree with @andrewabest. Additional note is that team references can create a view of struggle with those on solo gigs.

**Comment by kjacobsen** [comment]
Quick question, what is the issue these changes are resolving?

File: Consulting.md [hunk 6]
```diff
@@ -61,11 +63,16 @@
 - I adapt quickly to changing requirements and typically structure my approach to cater for this.
 - I have proven I can decompose larger requirements into smaller, more manageable pieces, to deliver functionality in an incremental and iterative manner.
 - I possess writing skills that let me be clear, concise, articulate and persuasive in all of my outgoing communications.
+- I Interact well with my team mates and our customers.
```
Comment (kjacobsen) [inline Consulting.md]: That probably should be:
> I interact well with my team mates and our customers.

File: Consulting.md [hunk 7]
```diff
@@ -61,11 +63,16 @@
...
+- My team find me easy to work with.
+- I participate in improving Readify processes.
+- I readily share my knowledge with fellow Readifarians and client developers.
```
Comment (kjacobsen) [inline Consulting.md]: How about changing this to be something less developer centric.

> I readily share my knowledge with fellow Readifarians and out clients technical teams

Perhaps someone has a better way of putting it.

**Comment by kjacobsen** [comment]
Additional text to consider from Senior Sustained Systems Engineer:

> I go out of my way develop a friendly and professional rapport with my customer and other team members.
> My contributions to team discussions are net-positive, informative and helpful.

**Comment by andrewabest** [comment]
Hi @mandyfegredo are you keen to pursue the above discussions on this PR? If not we can close it for now, and it can be reopened once there is time to devote to it.

**Comment by rbanks54** [comment]
It's gone stale. Closing.

**Pull request closed without merging**